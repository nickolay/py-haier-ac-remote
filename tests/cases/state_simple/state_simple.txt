00000000: 00 00 27 15 00 00 00 00  00 00 00 00 00 00 00 00  ..'.............
00000010: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  ................
00000020: 00 00 00 00 00 00 00 00  30 34 46 41 38 33 38 46  ........04FA838F
00000030: 38 34 35 41 00 00 00 00  00 00 00 00 00 00 00 00  845A............
00000040: 00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 25  ...............%
00000050: FF FF 22 00 00 00 00 00  01 06 6D 01 00 15 00 25  ..".......m....%
00000060: 00 7F 00 00 00 00 00 02  00 00 00 03 00 18 00 00  ................
00000070: 00 00 00 09 76                                    ....v