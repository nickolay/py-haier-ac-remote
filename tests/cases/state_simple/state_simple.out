State(current_temp=21, target_temp=25, fan_speed=<FanSpeed.MAX: 0>, mode=<Mode.HEAT: 2>, health=False, limits=<Limits.UNKNOWN_3: 3>, power=False)